### ISSUE TITLE
[A11Y] Unable to scroll the page using "Up/Down" arrow and "PgUp/PgDown" keys after reaching the end of the report

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#3481]
**Browser / Version**: Firefox Nightly 82.0a1 (2020-09-17)
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Fill in all required data from the form until reaching "Report anonymously" button. 
3. Use one "Tab" key.
4. Try to move up to top of the report ("Page up" button or "Up" arrow key).
5. Observe behavior.

**Expected Behavior:**
Page is scrolled up.

**Actual Behavior:**
Nothing happens, page is not scrolled up.

**Notes:**
1. The issue is not reproducible on Chrome 85.0.4183.102 

**Watchers:**
@softvision-oana-arbuzov
@cipriansv
